packages/calcite-components/src/components/action/action.tsx

```diff
@@ -309,10 +322,12 @@ export class Action extends LitElement implements InteractiveComponent {
 
     return (
       <button
-        aria-controls={indicator ? indicatorId : null}
-        ariaBusy={loading}
-        ariaLabel={ariaLabel}
-        ariaPressed={active}
+        ariaBusy={this.aria?.ariaBusy ?? loading}
+        ariaControlsElements={ariaControlsElements}
+        ariaDescribedByElements={this.aria?.ariaDescribedByElements}
+        ariaExpanded={this.aria?.ariaExpanded}
+        ariaLabel={this.aria?.ariaLabel ?? ariaLabel}
+        ariaPressed={this.aria?.ariaPressed}
```

Comment:
@jcfranco I'm not able to spread all the aria properties in here. I'm not sure if its better to only allow some aria properties or if we should allow them all which gives them more control. If we do want to allow all of them it might be useful to have a util to handle that since I can't spread them in JSX.

Reply:
For now, we can start with a focused set of props and follow up with the util you suggested. Otherwise, from an API-perspective, it'll be confusing if type allows all props, but only some are applied.

For the util case, I _think_ we'll need to block specific props/attributes to avoid breaking ARIA configurations we manage internally.